orchard/orchard_java_runner.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2022, Salesforce, Inc.
+// All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+// For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+package orchard
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"mce.salesforce.com/sprinkler/common"
+)
+
+type OrchardJavaRunner struct{}
```

Comment:
let's not support java here, the reason why I make it a generic command in the first place is that sprinkler should not care what it's running other than it's running a command that would produce a valid orchard workflow definition to the stdout

Reply:
okay.  I will refactor accordingly.

Reply:
java support taken out.  stdout runner refactored.